tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/GitHubService.cs

```diff
@@ -79,6 +79,8 @@ public interface IGitHubService
         public Task<PullRequest?> GetPullRequestForBranchAsync(string repoOwner, string repoName, string remoteBranch);
         public Task<Issue> GetIssueAsync(string repoOwner, string repoName, int issueNumber);
         public Task<IReadOnlyList<RepositoryContent>?> GetContentsAsync(string owner, string repoName, string path);
+        public Task UpdatePullRequestAsync(string repoOwner, string repoName, int pullRequestNumber, string title, string body, ItemState state);
+        public Task AddReleasePlanInfoInSdkAsync(string repoOwner, string repoName, int prNumber, string releasePlanLink, string specPrLink, string workItemLink, string apiVersion);
```

Comment:
This method should not be in GitHub service. GitHub service should have a method to update pull request description. APIs in GitHub service is just helper to send request to GitHub.